generate-catalog/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "generate-catalog"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+color-eyre = "0.6.3"
+ego-tree = "0.6.2"
+lazy_static = "1.5.0"
+regex = "1.10.5"
+reqwest = "0.12.5"
+scraper = "0.19.0"
+structopt = "0.3.26"
```

Comment:
This library is in maintenance mode and depends on clap v2. It's probably fine, but I'd suggest using [`pico-args`](https://docs.rs/pico-args) instead.